core/esmf-aspect-meta-model-python/README.md

```diff
@@ -1,84 +1,141 @@
+    * [Download test models](#download-test-models)
+  * [Aspect Meta Model Loader usage](#aspect-meta-model-loader-usage)
+  * [Samm Units](#samm-units)
+  * [SAMM CLI wrapper](#samm-cli-wrapper)
+* [Scripts](#scripts)
+* [Automation Tasks](#automation-tasks)
+  * [tox](#tox)
+  * [GitHub actions](#github-actions)
+    * [Check New Pull Request](#check-new-pull-request)
+    * [Build release](#build-release)
+<!-- TOC -->
+
 # An Aspect of the Meta Model
 
 The `esmf-aspect-model-loader` package provides the Python implementation for the SAMM Aspect Meta Model, or SAMM.
 Each Meta Model element and each Characteristic class is represented by an interface with a corresponding
 implementation.
 
-## Usage
+## Set Up SAMM Aspect Meta Model
 
-An Aspect of the Meta Model can be created as follows using the provided `AspectInstantiator`.
+Before getting started to use the `esmf-aspect-model-loader` library you need to apply some set up actions:
 
-```
-aspect_loader = AspectLoader()
-model_elements = aspect_loader.load_aspect_model("absolute/path/to/turtle.ttl");
-aspect = model_elements[0]
-```
+Required
+- [Install poetry](#install-poetry)
+- [Install project dependencies](#install-project-dependencies)
+- [Download SAMM files](#download-samm-files)
+Optional
+- [Download SAMM CLI](#download-samm-cli) (need to use SAMM CLI functions)
+- [Download SAMM test models](#download-test-models) (for running integration tests)
 
-or
+### Install poetry
 
-```
-aspect_loader = AspectLoader()
-aspect_urn = "urn:samm:org.eclipse.esmf.samm:aspect.model:0.0.1#AspectName"
-aspect = aspect_loader.load_aspect_model("absolute/path/to/turtle.ttl", aspect_urn);
+`Poetry` used as a dependency management for the `esmf-aspect-model-loader`. Follow the next [instruction](https://python-poetry.org/docs/#installation)
+ to install it.
+
+To check the poetry version run:
+```console
+poetry --version
 ```
 
-## Automatic Deployment
+### Install project dependencies
 
-A [GitHub action called 'Release'](https://github.com/eclipse-esmf/esmf-sdk-py-aspect-model-loader/actions/workflows/tagged_release.yml)
-has been set up for the `esmf-aspect-model-loader`. This action checks the code quality by running tests, the [static type checker MyPy](https://github.com/python/mypy) and
-the [code formatter 'Black'](https://github.com/psf/black).
+Poetry provides convenient functionality for working with dependencies in the project. 
+To automatically download and install all the necessary libraries, just run one command
+```console
+poetry install
+```
 
-## Set Up SAMM Aspect Meta Model for development
+### Download SAMM files
 
-In order to download the SAMM sources, it is required to run `poetry install` once in the `esmf-aspect-model-loader`
-module. There are two possibilities to download the SAMM files and extract the Turtle sources for the Meta Model.
+There are two possibilities to download the SAMM files and extract the Turtle sources for the Meta Model: 
```

Comment:
I think this section should also explain for whom are the SAMM files interesting, or in other words, why would you want to download them. IMO users of the aspect-meta-model-python should not have to deal with this at all (if they don't want to), including that they should not need to know about this.